src/pages/ReportSettingsPage.js

```diff
@@ -209,7 +209,7 @@ class ReportSettingsPage extends Component {
                                             onPress={this.validateAndUpdatePolicyRoomName}
                                             style={[styles.ml2, styles.mnw25]}
                                             textStyles={[styles.label]}
-                                            innerStyles={[styles.ph5]}
+                                            innerStyles={[styles.ph5, styles.saveButtonPadding]}
```

Comment:
I don't feel super strongly about this. But this style caught my eye as very specific. And also overwrites the "horizontal" part of `ph5`. If `ph5` adds a `paddingLeft` should we maybe just update this to be:

```js
saveButtonPadding : {
    paddingLeft: 20, 
    paddingRight: 18,
}
```

Reply:
Actually, why wouldn't we have equal horizontal padding on this one?

Reply:
we effectively do - but maybe there's a better way of doing this. Basically I changed the `buttonLarge` style across the board to NOT have equal padding, and this readjusts it to be 18px on each side (vs. 18 left 10 right). Good call that this overrides `ph5` - I think that was actually already happening, which is weird.

Reply:
Okay cleaned this up a bit - removed ph5, and added the `paddingLeft` into the `saveButtonPadding` style so it's clearer that they're equal.